### ISSUE TITLE
Usage

### ISSUE THREAD

**Issue by farawaytree** [johnsonjh/altoschat#3]
Was this used on the Altger Altos (from Germany?) of the late 80's? I think it ran on Unix...

The one that had multi-user chat where hackers congregated from all over the planet.

Those days are lost in history and not documented anywhere!

Ah yes I see the uucp... !mcvax!unido!altger!korn

That alone brings back HUGE memories.

Pengo... Korn... MH... camelot...

**Comment by johnsonjh** [comment]
Yes, this is the same one that ran on altger.  I believe it was mentioned in 1997 book "[Underground](https://www.underground-book.net/)" by Suelette Dreyfus, which is probably worth reading, if you haven't already.

**Comment by johnsonjh** [comment]
I was a very occasional user as 'trn' (which was short for The Raven's Nest, which was a BBS I ran way back when), so it is not my code - I only archived it here to ensure it stays available.  While I wouldn't want to run this talker on anything modern, but it sure might be fun to get it setup on the period-correct Xenix system some day.

**Comment by farawaytree** [comment]
Much appreciated. This history must be kept alive, because it really was the hacker 'golden era'.

I recall telnet'ing into altger, the chat was packed full of hackers back then. But next came the big US government bust late '88 (run by the secret service for some reason) which bought down MH and camelot... I later connected back in '94 from University and it was still running!

**Comment by johnsonjh** [comment]
I *think* Altger went down in 1995, but don't quote me on that.

Lutzifer went down in late 1993.  Phrack 45 says "December 20, 1993".  There was QSD (in France, minitel I believe) which was supposedly still around in 1995 or 1996.

There were a few other talkers and BBS systems over X.25 but I can't remember them all now.  There were a couple other talkers still around at even later stages, but ICB and IRC had mostly taken over by 1995, by my memories.  Some talkers lasted far into the internet era via telnet, but most were pretty lame.

Not directly X.25 related, but I *do* sort of miss the MajorBBS chats, which had features like being able to play dice and poker.  There is https://www.mbbsemu.com/ which helps to run most of the games that were available for it.

**Comment by johnsonjh** [comment]
I remember getting a pretty cool script from someone on altger that did X.25 NUA scanning from a PR1ME host.  Too bad I lost it.  Julian Assange used it too, as `prof` IIRC, before he was (in)famous.

**Comment by farawaytree** [comment]
Ah, that was probably me that sent you that if it were around 1988... I wrote an NUA scanner that I used to run on Surrey University Primes (via the Janet X.25 g/w). Probably written in CPL... can't remember now!

**Comment by johnsonjh** [comment]
It would have been later than 88 but earlier than 95!  Hard to remember exactly :)
